src/content/docs/en/tutorial/6-islands/3.mdx

```diff
@@ -57,6 +57,8 @@ Welcome to the universe, astronaut. 👩🏼‍🚀👨🏿‍🚀🧑‍🚀
 
 ## Next Steps
 
+[Migrate this project to content collections](/en/guides/content-collections/#migrating-from-file-based-routing)
```

Comment:
```suggestion
Continue with our [guide to migrate this project to content collections](/en/guides/content-collections/#migrating-from-file-based-routing)
```
I'm wondering whether explicitly calling out this is a GUIDE is helpful? Might feel more new-user friendly? (And, I think this emphasizes that the *this project* is the subject of that guide, vs "Now you can migrate this, like any project, to content collections")